src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -715,6 +715,24 @@ define('diagramStore', ['jquery', 'xwiki-meta', 'draw.io', 'xwiki-events-bridge'
     oldImportFiles.apply(this, newArgs);
   }
 
+  //
+  // Change the URL generated by export for containing the actual host.
+  //
+  var oldCreateLink = EditorUi.prototype.createLink;
+  EditorUi.prototype.createLink = function() {
+    let rawURL = oldCreateLink.apply(this, arguments);
+    var docURL = new XWiki.Document('Diagram', 'DiagramViewSheet').getURL();
```

Comment:
The pane name is the first argument (in order to have the space and wiki parameters optional; optional parameters are put at the end so that you don't have to specify them).
```suggestion
    var docURL = new XWiki.Document('DiagramViewSheet', 'Diagram').getURL();
```